<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectList</class>
 <widget class="QWidget" name="ObjectList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>448</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>300</height>
   </size>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="FilterByName">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="SearchBoxLabel">
       <property name="text">
        <string>Filter by name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="SearchBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="SimpleFilters">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="FilterTypeLabel">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="FilterType"/>
     </item>
     <item>
      <widget class="Line" name="HLine">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="CatalogListLabel">
       <property name="text">
        <string>Catalog:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="CatalogList"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="AdvancedSearch" native="true">
     <widget class="KPushButton" name="ShowAdvancedButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>145</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Advanced Search</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Magnitude">
     <item>
      <widget class="QLabel" name="LowerLimitLabel">
       <property name="text">
        <string>Magnitude between</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="MinMagnitude"/>
     </item>
     <item>
      <widget class="QLabel" name="UpperLimitLabel">
       <property name="text">
        <string>and</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="MaxMagnitude"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Semiaxis">
     <item>
      <widget class="QLabel" name="SemimajorAxisLabel">
       <property name="text">
        <string>Semimajor axis (a):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="SemimajorAxis"/>
     </item>
     <item>
      <widget class="QLabel" name="SemiminorAxisLabel">
       <property name="text">
        <string>Semiminor axis (b):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="SeminorAxis"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="AdvancedSearchLayout"/>
   </item>
   <item>
    <widget class="QWidget" name="SimpleSearch" native="true"/>
   </item>
   <item>
    <widget class="KSObjectList" name="TableView" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KSObjectList</class>
   <extends>QWidget</extends>
   <header>ksobjectlist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
